test/unit/utils/url.test.js

```diff
@@ -81,10 +80,7 @@ QUnit.test('isCrossOrigin can identify cross origin urls', function(assert) {
   // we cannot test that relative urls work on https, though
   assert.ok(!Url.isCrossOrigin('example.vtt'), 'relative url is not cross origin');
 
-  const location = {
-    protocol: 'https:',
-    host: 'google.com'
-  };
+  const location = new URL('https:/google.com');
 
   assert.ok(!Url.isCrossOrigin('https://google.com/example.vtt', location), 'http://google.com from https://google.com is not cross origin');
```

Comment:
LTGM. But while you're at it, would it be possible to modify the description of this test? If not, that's fine too.

```suggestion
  assert.ok(!Url.isCrossOrigin('https://google.com/example.vtt', location), 'https://google.com is not cross origin');
```